src/bp/ui-studio/src/web/actions/index.ts

```diff
@@ -169,18 +170,20 @@ export const duplicateFlow: (flow: { flowNameToDuplicate: string; name: string }
   }
 )
 
-export const requestRefreshParentFlow = createAction('FLOWS/REFRESH_PARENT_FLOW')
+export const requestRefreshCallerFlows = createAction('FLOWS/REFRESH_PARENT_FLOW')
 
-const updateParentFlow = async (_payload, state) => {
-  const flowState = state.flows
-  const parentFlow = flowState.flowsByName[flowState.currentFlow].parent
+const updateCallerFlows = async (_payload, state) => {
+  const flows = <FlowReducer>state.flows
+  const callerFlows = Object.values(flows.flowsByName).filter(x => x.nodes.find(n => n.flow === flows.currentFlow))
```

Comment:
Can you check to replace this with a selector? The value is computed only when something is changed, and we can reuse it. It's in `selectors.ts`, look at getCurrentFlowSubFlows.